deploy/src/main/deploy/helm/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: ${project.version}
+description: A Helm chart for Eclipse Hono
+name: eclipse-hono
+version: ${project.version}
```

Comment:
That's chart version. In theory app and chart can have different development cycles and don't need to have the same versions

Reply:
Do you think that would make sense? In the end we are talking about the Hono chart, aren't we?
What would be the meaning of a chart version 1.0.5 having an app version of e.g. 0.9.3? Wouldn't that be very confusing?

Reply:
Those are Helm concepts. I guess it could make sense to keep then out of sync in some cases, but as you said for us it's probably be the best to be synced (as they are now)

Reply:
Can we then at least comments to the `Chart.yaml` file explaining these properties (as you just did to me)? IMHO that would be helpful ...

Reply:
Done.